yamlpath/merger/mergerconfig.py

```diff
@@ -334,8 +346,15 @@ def _load_config(self) -> None:
 
         if config_file:
```

Comment:
This is just a case of not fixing an indent after making a change. Everything should work just fine be un-indenting those lines because even if the config file doesn't exist, we still use ConfigParser, adding it as a class member only if we added the overrides to it (by using `sections()`).

Though your comment made me realize that my tests should have caught this earlier, so I checked out my tests and realized that anything specified in `self.args` has a higher priority than my changes, which I think misses the mark. I'll take a deeper look into fixing this when I have a bit more time to update everything, but I'll push some changes to the tests soon to illustrate the issue.

Reply:
Disregard that last part, turns out it works just fine based on the tests I added. I added a fix and some tests for the fix.